front/bundlesize.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": [
+    {
+      "path": "./build/main.*.min.js",
+      "maxSize": "600 kB"
```

Comment:
Do we want to be a bit more lenient with this? Increase it a bit? Or do you think this is good enough?

Reply:
The main entrypoint is now about 550 kB. This is already much better than before, but still much higher than what's recommended (which is something like 240 kB even before gzip!). So I think it's good we keep a close eye on this. If we do find that it's too low we can always increase it by changing this single line of code